package.json

```diff
@@ -21,17 +22,19 @@
     "expo-updates": "~0.5.4",
     "react": "16.13.1",
     "react-dom": "^16.13.1",
-    "react-native": "0.63.2",
+    "react-native": "~0.63.4",
     "react-native-gesture-handler": "~1.10.2",
     "react-native-reanimated": "~2.1.0",
     "react-native-screens": "~3.0.0",
+    "react-native-snap-carousel": "^3.9.1",
     "react-native-unimodules": "~0.13.3",
     "react-native-web": "^0.16.3",
     "react-redux": "^7.2.4",
     "react-test-renderer": "^17.0.2",
     "redux": "^4.1.0",
     "redux-thunk": "^2.3.0",
-    "styled-components": "^5.3.0"
+    "styled-components": "^5.3.0",
+    "expo-status-bar": "~1.0.4"
```

Comment:
When do you use expo status bar?
If you don't use it uninstall it
npm rm expo-status-bar